lib/project_types/extension/tasks/find_npm_packages.rb

```diff
@@ -84,11 +84,11 @@ def list_packages
       end
 
       def yarn_list
-        production_only? ? %w[list --production] : %w[list]
+        production_only? ? %w[list --production --depth=0] : %w[list]
       end
 
       def npm_list
-        production_only? ? %w[list --prod --depth=1] : %w[list --depth=1]
```

Comment:
I believe the reason why `depth=1` was introduced was when we needed to know the version of the `cli` package for backwards compatibility, but we're no longer making a check on the cli package version, so I think it's ok to use `depth=0`.